### PR TITLE
CI: fix gtk-rs checker

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
WIP & depends on https://github.com/gtk-rs/checker/pull/13 being in first

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -116,17 +116,32 @@ jobs:
...
       - uses: actions/checkout@v2
         with:
           repository: gtk-rs/checker
           ref: master
           path: checker
-      - working-directory: checker
-        run: cargo build --release
-      - run: "./checker/target/release/checker ."
-      - run: "./checker/check_init_asserts"
+      - run: cargo build --release
```
Comment (GuillaumeGomez) [inline .github/workflows/CI.yml:144]: The `checker` is doing a lot of things, not just checking that it builds fine. So please don't remove it? :)

Comment (bilelmoussaoui) [reply .github/workflows/CI.yml:144]: No I didn't remove it ^^  the check_init_asserts is disabled for now until I fix the checker (it's still broken)

**Comment by bilelmoussaoui** [comment]
Hmm seems like something is not working properly with the check_init_assert script

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**